### DISCUSSION TITLE
QEMU error

### DISCUSSION THREAD

**Discussion by Lukerc9** [utmapp/UTM#6576] [UTM Help (macOS)]
UTM latest version- Macbook pro chip 3 sonoma- UTM with kali linux 

QEMU error: QEMU exited from an error: qemu-aarch64-softmmu: -drive if=none,media=cdrom,id=driveB489B557-7154-4441-AD4B-BEB8F41EF96D/Downloads/kali-linux-2024.1-installer-arm64.iso,readonly=on: Failed to lock byte 100 

This was the error that occured when i tried to boot up my vm for the first time. New user to this and i saw in the past people had similar issues but couldnt find the resolution.

**Comment by Lukerc9** [comment]
Never. mind fixed it

**Comment by noynek** [comment]
What was the solution? Seeing the same issue.

**Comment by osy** [comment]
Next person to hit this problem, please open a new issue and attach debug.log
